app/main.py

```diff
@@ -1,13 +1,30 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {"line": error["line_number"],
+            "column": error["column_number"],
+            "message": error["text"],
+            "name": error["code"],
+            "source": "flake8"
+            }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {"errors": [{"line": errors[line]["line_number"],
```

Comment:
reformat your code with these suggestions:
 https://github.com/mate-academy/py-linter-formatter/blob/master/checklist.md
 
for this lines - don't duplicate your code, use existed functions